services/provider/server/consumer.go

```diff
@@ -211,24 +212,37 @@ func (c *ocsConsumerManager) UpdateStatusLastHeatbeat(ctx context.Context, id st
 	}
 	c.mutex.RUnlock()
 
-	patchInfo := struct {
+	patchInfo := []struct {
```

Comment:
Why it is so important for us to patch instead of just reading and re-writing the status?

Reply:
done.